models/user.model.js

```diff
@@ -6,6 +6,11 @@ const UserSchema = mongoose.Schema({
 		type: String,
 		required: true,
 	},
+	username: {
```

Comment:
Is 'username' part of the model? @iamNanaKwame @TwumasiAckah @hadjakade

Reply:
Yes. username is part.
@iamNanaKwame are we ignoring usernames?